examples/CustomEntitlementComputationSample/app/src/main/java/com/revenuecat/sample/main/MainState.kt

```diff
@@ -0,0 +1,13 @@
+package com.revenuecat.sample.main
+
+import com.revenuecat.purchases.CustomerInfo
+import com.revenuecat.purchases.Offerings
+
+data class MainState(
+    val currentCustomerInfo: CustomerInfo? = null,
+    val displayErrorMessage: String? = null,
+    val shouldNavigateToPaywall: Boolean = false,
+    val shouldStartSwitchingUser: Boolean = false,
+    val shouldStartShowingBottomSheet: Boolean = false,
```

Comment:
Also unused, but maybe planned for another PR?

Reply:
These are from my attempts to show the infomation around the app what the app is for

Reply:
FWIW in SwiftUI I'd usually name these `shouldShowBottomSheet` (i.e.: a status rather than an action like start), since this will be shown as long as the flag is there, it's not really a _start_ of something

Reply:
Good point, I like that better, I will rename `shouldStartSwitchingUser`